source/query/filter.txt

```diff
@@ -364,6 +364,35 @@ The query returns the following document:
       "dateCreated": { "$date": "2003-03-26" }
    }
 
+Match by Nested Field
+~~~~~~~~~~~~~~~~~~~~~
+
+This example performs the following operations:
+
+- Adds a document with ``first`` and ``last`` subfields nested in the
+  ``name`` field
+- Finds the document with the ``last`` subfield of "Washington"
+
+Add the following document to your collection:
+
+.. code-block:: shell
+
+   {
```

Comment:
Rather than adding a new document, could we maybe instead add an embedded field to the existing data at the top of the Examples section? The intro to the Examples section states that dataset can be used for the examples, so it might be confusing to have one example that uses a different sample doc.

We could keep it simple and do something like:

```
{
"name": "Andrea Le",
"email": "andrea_le@fake-mail.com",
"school": {
   "name": "NYU"
}
"version": 5,
"scores": [ 85, 95, 75 ],
"dateCreated": { "$date": "2003-03-26" }
}